### PR TITLE
STAC-12223 Update process-agent with the version with HTTP metrics

### PR DESCRIPTION
**Pull request by Q69K**

### CODE REVIEW

File: pkg/config/config_template.yaml [hunk 1]
```diff
@@ -806,10 +806,25 @@ network_tracer_config:
 # Network tracer specific settings
 #
 # network_tracer_config:
-#   A boolean indicating the enabled state of the network tracer.
-#   enabled: true
-#   The full path to the file where network-tracer logs will be written.
+#   # A boolean indicating the enabled state of the network tracer.
+#   network_tracing_enabled: "true"
```
Comment (hierynomus) [inline pkg/config/config_template.yaml:810]: Why is this renamed, and why is it quoted (instead of a real yaml bool)

Comment (julianosk) [reply pkg/config/config_template.yaml:810]: All flags are string to support "yes", "no".. and the renaming is a fix, in `stackstate-process-agent` this is the name of the flag.

**Review by hierynomus — APPROVED** [review]
